src/dorsal.js

```diff
@@ -355,12 +359,11 @@ DorsalCore.prototype.unwire = function(el, pluginName) {
  */
 DorsalCore.prototype.wire = function(el, pluginName) {
     var deferred = new DorsalDeferred(this.ELEMENT_TO_PLUGINS_MAP),
-        pluginKeys = this.registeredPlugins(),
-        responses,
+        responses = [],
         action;
 
     if (!this.plugins) {
-        throw new Error('No plugins registered with Dorsal');
+        console.warn('No plugins registered with Dorsal');
```

Comment:
I will add an `if (console && console.warn) ` to it. 

I'm +0 with this change, but if some of the other maintainers agreed. we can merge it.